packages/gravity-ui-web/gulp/color-scheme-tables.js

```diff
@@ -0,0 +1,66 @@
+const Color = require('color');
+const kebabCase = require('lodash.kebabcase');
+const { colorSchemes } = require('@buildit/gravity-particles');
+
+
+
+function wcagRating(contrastRatio) {
+  if (contrastRatio > 7) {
+    return 'AAA';
+  }
+  if (contrastRatio > 4.5) {
+    return 'AA';
+  }
+  if (contrastRatio > 3) {
+    return 'AA-lrg'; // AA for large text (>-18pt) and UI elements
+  }
+  return 'Fail';
+}
+
+function gravityCssCustomPropName(group, colorName) {
+  return `--grav-co-grp-${group}-${kebabCase(colorName)}`;
+}
+
+
+function generateColorSchemeTableData(colorScheme) {
+  const groupAColorNames = Object.keys(colorScheme.groupA);
+  const groupBColorNames = Object.keys(colorScheme.groupB);
+
+  const tableData = [];
+  const headerRow = [null].concat(groupAColorNames.map((name) => gravityCssCustomPropName('a', name))); // top-left cell is empty
```

Comment:
This looks really weird to me since the array has a null element in it to start with. Thinking about it, it makes sense since the first row should be empty. Might be a strong case for the table being a `class` instead of a returned object due to complexity of the generated table.